howso/engine/trainee.py

```diff
@@ -2911,10 +2911,10 @@ def react_into_trainee(
         Parameters
         ----------
         action_feature : str, optional
-            Name of target feature whose hyperparameters to use
-            for computations.  Default is whatever the model was analyzed for,
-            or the mda_action_features for MDA, or ".targetless" if analyzed
-            for targetless.
+            Name of target feature for which to do computations. Default is
+            whatever the model was analyzed for, i.e., action feature for MDA
```

Comment:
Technically I think this would be best to be an `e.g.` rather than an `i.e.` but that's very nitpicky

Reply:
eh, I can do it.